### PR TITLE
Fix yml #15 [semver:patch]

### PR DESCRIPTION
**Pull request by RonnyTite**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [ ] Examples have been added for any significant new features
- [ ] README has been updated, if necessary

### Motivation, issues

This PR is fixing #15 

### Description

 
 
 The fix being to replace:
```
apt-get update
```
by
```
apt-get --allow-releaseinfo-change update
```

`--allow-releaseinfo-change` description :

> Allow the update command to continue downloading data from a repository which changed its information of the release contained in the repository indicating e.g a new major release. APT will fail at the update command for such repositories until the change is confirmed to ensure the user is prepared for the change.

Source: https://manpages.debian.org/unstable/apt/apt-get.8.en.html

### CODE REVIEW

File: src/commands/install.yml [hunk 1]
```diff
@@ -17,7 +17,9 @@ steps:
 
         if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
 
-        $SUDO apt-get update && $SUDO apt-get -qqy install apt-transport-https
+        # https://github.com/CircleCI-Public/azure-cli-orb/issues/15
+        # https://manpages.debian.org/unstable/apt/apt-get.8.en.html
+        $SUDO apt-get --allow-releaseinfo-change update && $SUDO apt-get -qqy install apt-transport-https
```
Comment (EricRibeiro) [inline src/commands/install.yml:22]: While this works, I believe it's better to be specific about what we allow. This way, we don't hide unexpected changes. That said, what do you think of replacing `--allow-releaseinfo-change-suite` with `--allow-releaseinfo-change`?

You can read more about it here: https://stackoverflow.com/a/68990841.

File: src/commands/install.yml [hunk 2]
```diff
@@ -43,7 +45,9 @@ steps:
 
         # Update and install the Azure CLI
 
-        $SUDO apt-get update
+        # https://github.com/CircleCI-Public/azure-cli-orb/issues/15
+        # https://manpages.debian.org/unstable/apt/apt-get.8.en.html
+        $SUDO apt-get --allow-releaseinfo-change update
```
Comment (EricRibeiro) [inline src/commands/install.yml:50]: Same as above.

**Review by EricRibeiro — CHANGES_REQUESTED** [review]
I appreciate the help!

**Comment by RonnyTite** [comment]
@EricRibeiroThanks for the feedback. Changes done ;) you can take a new look

**Comment by EricRibeiro** [comment]
One of the tests failed: https://app.circleci.com/pipelines/github/CircleCI-Public/azure-cli-orb/122/workflows/ac3c877b-3926-4a51-a68e-40146e1ab1b0/jobs/406?invite=true#step-101-50.

I'm not 100% sure without investigating but updating the [test's executor images](https://github.com/CircleCI-Public/azure-cli-orb/blob/master/.circleci/config.yml#L18) might do the trick. Both images used for testing this orb are [outdated](https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/). The newest versions are [cimg/go](https://circleci.com/developer/images/image/cimg/go) and [cimg/python](https://circleci.com/developer/images/image/cimg/python).

File: .circleci/config.yml [hunk 3]
```diff
@@ -76,7 +76,7 @@ jobs:
           detect-login-type: true
 
   test-orb-golang:
-    executor: golang
+    executor: go
```
Comment (EricRibeiro) [inline .circleci/config.yml:79]: If we are changing the argument we are passing here, we also need to change the executor's name from `golang` to `go`: https://github.com/CircleCI-Public/azure-cli-orb/blob/master/.circleci/config.yml#L19

**Review by EricRibeiro — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 4]
```diff
@@ -19,12 +19,12 @@ executors:
   golang:
     resource_class: small
     docker:
-      - image: circleci/golang
+      - image: cimg/go:1.18.0
```
Comment (EricRibeiro) [inline .circleci/config.yml:22]: The image tag here should be `1.18`. The tag `1.18.0` doesn't exist.

**Comment by EricRibeiro** [comment]
Thank you, @RonnyTite ✌️ 

It's all looking good now. We still need to update that golang image from the tests, but it's outside this PR's scope.

I'll push that change later.

**Review by EricRibeiro — APPROVED** [review]

**Comment by EricRibeiro** [comment]
v1.2.2 released with this fix: https://circleci.com/developer/orbs/orb/circleci/azure-cli?version=1.2.2.

**Pull request merged**